### PR TITLE
Add example import and export chart

### PR DESCRIPTION
**Pull request by markcocker**

### CODE REVIEW

File: _docs/community.md [hunk 1]
```diff
@@ -3,16 +3,19 @@
...
   - [Agile Price Table](#agile-price-table)
   - [Export Rates Chart](#export-rates-chart)
+  - [Import And Export Rates Chart](#import-and-export-rates-chart)
 
 These are a few contributions by the community.
 
 ## Agile Price Table
 
-If you're wanting to display upcoming prices in a nice readable format, then might I suggest you utilise the plugin developed by @lozzd available at https://github.com/lozzd/octopus-energy-rates-card.
+If you're wanting to display upcoming prices in a nice readable format, then might I suggest you utilise the [Octopus Engergy Rates card](https://github.com/lozzd/octopus-energy-rates-card).
```
Comment (BottlecapDave) [inline _docs/community.md:12]: Could you add the credit to @lozzd back in please?

File: _docs/community.md [hunk 2]
```diff
@@ -120,7 +124,120 @@ yaxis:
...
+
+## Import and Export Rates Chart
+
+If you're looking to combine import and export rates then create a card with the configuration:
+
+
+```yaml
+type: custom:config-template-card
+entities:
+  - sensor.octopus_energy_electricity_21l4726127_2000021792976_current_rate
```
Comment (BottlecapDave) [inline _docs/community.md:141]: you might want to remove the identifiers for your meters?

File: _docs/community.md (hunk 1 above)
Comment (markcocker) [reply _docs/community.md:12]: Done - removed by accident.

File: _docs/community.md (hunk 2 above)
Comment (markcocker) [reply _docs/community.md:141]: Done

**Comment by BottlecapDave** [comment]
Thanks for the contribution :)

**Pull request merged**